packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeDefinition.ts

```diff
@@ -77,14 +77,13 @@ export interface ElementsGroupInfo {
         modelIds: Id64String[];
         type: "category";
       };
-  className: string;
-  groupingNode: GroupingHierarchyNode;
+  groupingNode: ClassGroupingHierarchyNode;
 }
 
 interface ModelsTreeInstanceKeyPathsFromInstanceKeysProps {
   imodelAccess: ECClassHierarchyInspector & LimitingECSqlQueryExecutor;
   idsCache: ModelsTreeIdsCache;
-  keys: Array<InstanceKey | ElementsGroupInfo>;
+  focusedItems: Array<InstanceKey | ElementsGroupInfo>;
```

Comment:
I don't think this is a good rename. Paths lookup based on target items will probably be most often done due to focusing, but not necessarily. So we shouldn't use the notion of "focus" in this context. I'd change this to `targetItems` or `targets`. Also, I think the interface name should also be changed accordingly, e.g. `ModelsTreeInstanceKeyPathsFromTargetItemsProps`.

Reply:
Renamed to `targetItems` and interface to `ModelsTreeInstanceKeyPathsFromTargetItemsProps`